zamba/models/efficientnet_models.py

```diff
@@ -35,7 +35,6 @@ def __init__(
                 efficientnet.get_submodule("blocks.5"),
                 efficientnet.conv_head,
                 efficientnet.bn2,
-                efficientnet.act2,
```

Comment:
in the latest release of timm, the activation is now part of `bn2`

previously it looked like:
```
  (conv_head): Conv2d(328, 2152, kernel_size=(1, 1), stride=(1, 1), bias=False)
  (bn2): BatchNorm2d(2152, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
  (act2): SiLU(inplace=True)
  (global_pool): SelectAdaptivePool2d (pool_type=avg, flatten=Flatten(start_dim=1, end_dim=-1))
```
but now it looks like
```
  (conv_head): Conv2d(328, 2152, kernel_size=(1, 1), stride=(1, 1), bias=False)
  (bn2): BatchNormAct2d(
    2152, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True
    (drop): Identity()
    (act): SiLU(inplace=True)
  )
  (global_pool): SelectAdaptivePool2d (pool_type=avg, flatten=Flatten(start_dim=1, end_dim=-1))
```

Reply:
we could pin to a lower version of timm but this seems like a reasonable substitution